asynction/loader.py

```diff
@@ -0,0 +1,9 @@
+from importlib import import_module
+from typing import Callable
+
+
+def load_handler(handler_id: str) -> Callable:
```

Comment:
Would prefer this to remain under `server.py`, at least for now where the existing modules aren't that big.